### PR TITLE
Move: Put Celery code in the BookServer.

### PR DESCRIPTION
**Pull request by bjones1**
Stop supporting old web2py-based LP problems as part of the move.

### CODE REVIEW

**Comment by bjones1** [comment]
This PR requires https://github.com/RunestoneInteractive/BookServer/pull/102 to be merged first in order to pass.

**Comment by bnmnetp** [comment]
I merged BookServer#102 but this still fails...  

There is an error message about a path `None/bookserver` which seems pretty clearly wrong.

File: pyproject.toml [hunk 1]
```diff
@@ -25,7 +25,6 @@ altair = "^4.0.0"
 beautifulsoup4 = "^4.0.0"
 bleach = "> 3.1.1"
 bookserver = "^1.0.0"
-celery = { extras = ["redis,gevent"], version = "^5.0.0" }
```
Comment (bnmnetp) [inline pyproject.toml:28]: Shouldn't there be a poetry.lock file corresponding to this change to pyproject.toml ??

**Comment by bjones1** [comment]
Fixed now.

File: pyproject.toml (hunk 1 above)
Comment (bjones1) [reply pyproject.toml:28]: I added this -- thanks.

**Comment by bjones1** [comment]
This is a larger change, following up on PRs in the BookServer to fix how Celery works. I don't see easy ways to split this into smaller PRs.

**Review by bnmnetp — APPROVED** [review]

**Pull request merged**